tasks/push-rpm-manifests-to-pyxis/tests/test-push-rpm-manifests-to-pyxis.yaml

```diff
@@ -0,0 +1,102 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-push-rpm-manifests-to-pyxis
+spec:
+  description: |
+    Run the push-rpm-manifests-to-pyxis task with required parameters - a happy path scenario.
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-values
+            image: quay.io/redhat-appstudio/release-service-utils:8bf56a04aaeb371f4a822d2b76520e9bdcacfb26
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+
+              cat > $(workspaces.data.path)/pyxis_data.json << EOF
+              {
+                "components": [
+                  {
+                    "pyxisImages": [
+                      {
+                        "arch": "amd64",
+                        "imageId": "myImageID1",
+                        "digest": "mydigest1",
+                        "arch_digest": "abcdefg",
+                        "containerImage": "imageurl1"
```

Comment:
Maybe use a more realistic url in at least one instance to check that the `tr` substitution works as expected?

Reply:
Updated the second image in this test

Reply:
Could you make it even more realistic please? ;) Like `"quay.io/org/repo@sha256:0123456abcdef"`

Reply:
Sure thing